components/automate-ui/src/app/pages/top-nav-landing/top-nav-landing.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component } from '@angular/core';
+import { RoutePerms } from 'app/components/landing/landing.component';
+import { LayoutFacadeService } from 'app/entities/layout/layout.facade';
+
+@Component({
+  selector: 'app-top-nav-landing',
+  templateUrl: './top-nav-landing.component.html',
+  styleUrls: ['./top-nav-landing.component.scss']
+})
+export class TopNavLandingComponent {
+
+  // order must match layout in navbar.component.html
+  public routeList: RoutePerms[] = [
+    {
+      allOfCheck: [
```

Comment:
Since we're doing microoptimizations, can we ditch the third element that always is `''`?

Reply:
Good point; done.